### ISSUE TITLE
Review Questions 1479 and 2968 about IUU or split into multiple questions 

### ISSUE THREAD

**Issue by serenalomonico** [shcaba/FishPath-updates#23]
1479: Is high discarding or illegal, unregulated, or unreported activity occurring or suspected?

In Conil, the FishPath Team noted to either split this into 2 questions or clarify wording, because high discards and IUU can be 2 very different things.

**Comment by serenalomonico** [comment]
Additional comment by Natalie: IUU questions should perhaps have response options teased out. There is a difference between minor misreporting here and there, and more serious IUU behaviour.

Comments on splitting up IUU Questions from NSW:
●  	There are a number of fisheries that may have, within the broad context of IUU, non-compliance that wouldn’t taint representative sampling data because the offending is technical. To suggest that those fisheries (or species) are impacted through IUU could be misleading and may sway an assessment into being too precautionary, potentially at a negative cost to industry or other sectors.
●      IUU could considered in terms of minor or serious non-compliance.
●      For example, Q17 in Monitoring could be rephrased to
●      Is the non-compliance affecting the fishery/species, of a technical or minor nature?
●      A second question could then be included to identify the IUU component
●      Is the non-compliance of a level to be considered IUU under relevant Plans of Action?
●      If the response to the question(s) is yes/no, then the caveat provided should not necessarily suggest that sampling may not be representative.
●      If the response to both questions is no/yes, the existing caveat would be appropriate.
●      Additional questions could be considered about governance responses to IUU fishing activities, like using risk based compliance strategies, higher penalties for serious or recidivist offending and even internal deterrent systems such as demerit point programs to reduce the inference that monitoring programs are impacted because of IUU activity.
●      Where greater deterrence drives voluntary compliance, the caveat could be lessened because the impacts should be lessened on the fishery or species.
●      Perceptions about IUU and enforcement will vary across a diverse range of people and the impacts of those perceptions could mislead a FishPath assessment. By further defining the IUU reference (as most Australian fisheries compliance agencies do) better data and better decision making outcomes are possible.
●  	Other questions, across the assessment and decision rule pathways could reflect a similar type of enhancement where relevant.

**Comment by serenalomonico** [comment]
Also resolve this with Question 2968: Is the level of illegal, unregulated, or unreported fishing such that the stock or species of interest is affected to the extent that a data collection program would not be able to capture the extent of fishing mortality?

**Comment by dtdougherty** [comment]
On 3/11/19 task team call, we decided to merge the to questions into question 2968. We have transferred everything from question 1479 and now need to update management measure caveats for other measures that say 

"Depending on the extent of the illegal, unregulated, or unreported fishing, this may be a more appealing option that direct controls on catch or effort, but its effectiveness may be compromised."

Still need to delete question #1479

**Comment by bsnouffer** [comment]
I think there was a saving issue when this was done originally. After discussing with Natalie and Dawn, I moved the assessment caveats from 1479 to 2968. However, there are a number of new questions in review regarding IUU, enforcement and management measures. The caveats from 1479 will be reconciled with the new proposed questions and caveats. Once that is done, I will delete 1479.

**Comment by bsnouffer** [comment]
Finished the review of this based on the peer review comments and what came out of the enforcement question update. 1479 has been merged into 2968 (MM caveats and description) and 2968's text has been updated to:

> Is there a high level of illegal, unregulated, or unreported fishing, or the level of unreported discarding is unknown, such that the extent of fishing mortality is unable to be determined for the stock or species of interest?

Description now reads:

> This question seeks to identify sources of fishing mortality that might not be able to be quantified through data collection efforts by managing agencies and therefore would compromise the effectiveness of the assessment or of certain management measures.
In the context of this question, note that discarding may be legal - the question is referring to unreported sources of significant fishing mortality.

1479 has been deleted.
